src/Disp.js

```diff
@@ -1249,6 +1249,49 @@ CM.Disp.ReplaceTooltipLump = function() {
 	}
 };
 
+/**
+ * This function replaces the original Game.crate and Game.crateTooltip functions of stats page
+ */
+CM.Disp.ReplaceCrateTooltipAchievements = function() {
```

Comment:
This implementation does not really work, as it fully replaces the `Game.crate` and `Game.tooltip` functions instead of only sections. You will notice that if you load these functions CookieMonster will crash as a result of this.
We will therefore need to find a way to overwrite only a section of these functions without fully copying them into our own code.